src/material-examples/tooltip-auto-hide/tooltip-auto-hide-example.css

```diff
@@ -0,0 +1,11 @@
+button[matTooltip] {
```

Comment:
Make this an `.example-...` class. The goal to to ensure that people inspecting the elements know what the real styles are vs. the example styles.